api/v1alpha5/types.go

```diff
@@ -273,8 +273,8 @@ func (r SecurityGroupRule) Equal(x SecurityGroupRule) bool {
 type InstanceState string
 
 var (
-	// InstanceStateBuilding is the string representing an instance in a building state.
-	InstanceStateBuilding = InstanceState("BUILDING")
+	// InstanceStateBuild is the string representing an instance in a build state.
+	InstanceStateBuild = InstanceState("BUILD")
```

Comment:
While it is being created Nova server status [is BUILD and not BUILDING](https://docs.openstack.org/api-guide/compute/server_concepts.html#server-status). As this variable was not used elsewhere I felt it was ok to change it, please let me know it you foresee any issue to do that.

Reply:
VM state seems to be [derived from status](https://github.com/openstack/nova/blob/d4f34ab8d8ba255e2514b88f22d49f9968d22e16/nova/api/openstack/common.py#L115), and the correspondance is not staightforward, for BUILDING state, we have a [BUILD status](https://github.com/openstack/nova/blob/d4f34ab8d8ba255e2514b88f22d49f9968d22e16/nova/api/openstack/common.py#L65)